### ISSUE TITLE
Images not rendering correctly

### ISSUE THREAD

**Issue by Sasikumar3595** [dotnet/maui#5484]
### Description

We have created the application with images control and the images are not rendered correctly. The UI looked like a drawing issue.

```

        ScrollView customStackScrollview = new ScrollView();
        StackLayout customControlStack = new StackLayout();
        for (int i = 0; i < 10; i++)
        {
            Grid imageGrid = new Grid();
            imageGrid.Margin = new Thickness(5);
            imageGrid.HeightRequest = 100;
            Image image = new Image();
            image.Source = "march.png";
            image.Aspect = Aspect.Fill;
            Label label = new Label();
            label.Text = i.ToString();
            label.TextColor = Colors.Black;
            label.FontSize = 16;
            label.Padding = new Thickness(10);
            imageGrid.Add(image);
            imageGrid.Add(label);
            customControlStack.Add(imageGrid);
        }
        customStackScrollview.Content = customControlStack;
        this.Content = customStackScrollview;

```

**Image:**

<img width="1920" alt="march" src="https://user-images.githubusercontent.com/44917578/159450680-d99478b7-6b29-44bb-abd0-fa66e1e37db7.png">

**Output UI:**

![Issue](https://user-images.githubusercontent.com/44917578/159450740-a8f5bbd7-56af-4eba-89e7-3d7f329d7d8c.png)

### Steps to Reproduce

1. Create a sample with the above code
2. Add images to image folder under resources
3. Run the sample the images will not render correctly

### Version with bug

Preview 14 (current)

### Last version that worked well

Preview 13

### Affected platforms

iOS, Android, Windows, macOS

### Affected platform versions

Android 10

### Did you find any workaround?

No

### Relevant log output

```shell
Not Applicable
```

**Comment by XamlTest** [comment]
Verified not repro with Visual Studio Enterprise 2022 main 17.2.0 Preview 2.0 [32223.438.main] on Android. Sample Project: [5484.zip](https://github.com/dotnet/maui/files/8323399/5484.zip)

![image](https://user-images.githubusercontent.com/97647558/159458671-0b2eff99-9f0a-407e-bd9e-7b08e92e1ebd.png)

**Comment by XamlTest** [comment]
@Sasikumar3595 Can you try with my project? Maybe the issue is related to image size.

**Comment by Redth** [comment]
Should be fixed when #5198 is completed.

**Comment by davidortinau** [comment]
This is closed per #5198. If this continues to repro for you, please open a new issue with a specific sample.
